packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -110,9 +108,10 @@ class HotRunner extends ResidentRunner {
     // TODO(cbernaschina): check that isolateId is the id of the UI isolate.
     final OperationResult result = await restart(pause: pause);
     if (!result.isOk) {
-      throw rpc.RpcException(
-        rpc_error_code.INTERNAL_ERROR,
+      throw vm_service.RPCError(
         'Unable to reload sources',
+        -32603,
```

Comment:
What should we do about these constants? I could hoist them back into vmservice.dart, since we'll need to leave a few things there

Reply:
Thats what I did